postgresql/manifests/backup-pvc.yaml

```diff
@@ -0,0 +1,11 @@
+# PVC for backup volume
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: postgres-backup-pvc
+spec:
+  accessModes:
+  - ReadWriteOnce
```

Comment:
Nit: The indentation of list values is inconsistent (a personal pet peeve of mine with YAML).  In this file you're not indenting the `- ReadWriteOnce`, but in other files you are.  It'd be nice to be consistent.

Reply:
Run through yamlformat so it should now be consistent